docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,71 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+You can set up a local agent to execute pipelines launched in Dagster+. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
```

Comment:
I know I was the original author, but rereading this, "**Local** agents are a good way to experiment with Dagster+ **locally**" is a bit tautological 😄 - maybe we drop the second "locally?"